packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -27,6 +27,10 @@ import { Endpoint } from '@theia/core/lib/browser/endpoint';
 import { VSXEnvironment } from '../common/vsx-environment';
 import { VSXExtensionsSearchModel } from './vsx-extensions-search-model';
 import { VSXExtensionNamespaceAccess, VSXUser } from '../common/vsx-registry-types';
+import { CommandRegistry, MenuPath } from '@theia/core/lib/common';
+import { ContextMenuRenderer } from '@theia/core/lib/browser';
+
+export const MANAGE_EXTENSION_MENU: MenuPath = ['manage_extension_menu'];
```

Comment:
We should probably find a common place to put the `MenuPath`.
Additionally, we should create a sub-path for `copy` actions (so we can have the context-menu properly divided and categorized).